virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -22,6 +22,11 @@ When you create virtual machines from the web console, select a virtual machine
 
 Templates without a boot source are labeled as *Boot source required*. You can use these templates if you complete the steps for xref:../../virt/vm_templates/virt-creating-vm-template.adoc#virt-adding-a-boot-source-web_virt-creating-vm-template[adding a boot source to the virtual machine].
 
+[IMPORTANT]
+====
+You must remove the `evictionStrategy` from any templates or virtual machines that use `DataVolumes` and storage profiles to ensure compatibility with `SingleReplica` mode.
```

Comment:
Same here, to give context when this is relevant I would write it like this:

Due to differences in storage behavior, to ensure compatibility when creating a cluster with `SingleReplica` mode, you must remove the `evictionStrategy` from any templates or virtual machines that use `DataVolumes` and storage profiles.

Reply:
Updated admonition to reflect full text of bullet from Single Replica mode section.